<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tab_src</class>
 <widget class="QWidget" name="tab_src">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="MplCanvas" name="widget_mpl" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplCanvas</class>
   <extends>QWidget</extends>
   <header>GUI/ModifiedWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
